### PR TITLE
GIX-1101: Minor improvements SNS-1

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Final and minor improvements for SNS-1 launch.

# Changes

* Hide Committed projects title if no committed projects.
* Change message when no current launches.
* Add total amount of SNS tokens in Project Details page.
* Add new SnsAndCommunityFund topic to default topics.

# Tests

* Test new case in Launchpad.
* Test new info in ProjectSwapDetails.

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UI:

Screenshot 2022-10-31 at 08 06 52
Screenshot 2022-10-31 at 08 20 01

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

File: frontend/src/lib/components/launchpad/Projects.svelte [hunk 1]
```diff
@@ -46,7 +46,7 @@
     {/each}
   </div>
   {#if projects.length === 0}
-    <p class="no-projects">{noProjectsMessage}</p>
+    <p class="no-projects">{@html noProjectsMessage}</p>
```
Comment (peterpeterparker) [inline frontend/src/lib/components/launchpad/Projects.svelte:49]: As I double checked if it was safe to use `@html` here, can you rename `noProjectsMessage` into `noProjectsMessageLabel` or `noProjectsMessagei18n` just to make a bit more obvious its a static label?

Comment (lmuntaner) [reply frontend/src/lib/components/launchpad/Projects.svelte:49]: sure

File: frontend/src/lib/i18n/en.json [hunk 2]
```diff
@@ -538,7 +538,7 @@
     "committed_projects": "Launched",
     "no_committed_projects": "No launched projects at the moment",
     "open_projects": "Current Launches",
-    "no_open_projects": "No current launches at the moment",
+    "no_open_projects": "Looks like there's no SNSs launching right now. <a href=\"https://internetcomputer.org/sns\" rel=\"noopener noreferrer\" aria-label=\"more info about launching an SNS\" target=\"_blank\">More about creating SNSs.</a>",
```
Comment (peterpeterparker) [inline frontend/src/lib/i18n/en.json:541]: nth: not my mothertongue but maybe the call to action needs to be reviewed. e.g. "There is currently no ongoing SNS sales. Looking to start yours? Find more information here." or something

**Review by peterpeterparker — APPROVED** [review]
LGTM, minor comments. Thx

File: frontend/src/lib/i18n/en.json (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/lib/i18n/en.json:541]: Peter did this message...

https://dfinity.slack.com/archives/C048415NHTR/p1666965473616039

Comment (peterpeterparker) [reply frontend/src/lib/i18n/en.json:541]: Thanks. I assumed we rather liked a direct msg (as "Find out how to create an SNS") sentence, all good then.

**Pull request merged**